recipes/numba/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "numba" %}
+{% set version = "0.36.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a3b7540713c39f4d7a8d3b18cd49249c6356a35ee1ab5e15a89f7cf970004f0a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - llvmlite ==0.21.0
+    - numpy >=1.10
```

Comment:
So this should be a fixed `numpy` version in `build` if we are linking to it (e.g. `numpy 1.10.*`).

Reply:
I have a question, Would we allow users to install ``numba`` if numpy version is ``1.11`` or greater, since we are linking to ``1.10``?

Reply:
FWIW it does [appear that `numba` uses `numpy`'s data model]( https://github.com/numba/numba/blob/a4e6d6689d11ddad4125a01c4e4ad19bc69c5759/numba/npyufunc/_internal.h#L10 ). So we do have to treat `numba` as if it is linking to `numpy`.

Reply:
Sorry question appeared after the comment above was posted.

So this will only control what version of `numpy` we build against. We can use any version of `numpy` equal or greater than that at runtime.

Reply:
Does that make sense? Basically we have to fix `numpy` to `1.10.*` during `build` to allow users the flexibility of `numpy >=1.10` at runtime (in `run`).

Reply:
Ok. Got it